packages/data-context/src/sources/CloudDataSource.ts

```diff
@@ -129,7 +129,7 @@ export class CloudDataSource {
       return dfd.promise
     }
 
-    return pipe(executingQuery, toPromise).then((data) => {
+    return pipe(executingQuery, take(1), toPromise).then((data) => {
```

Comment:
Why do we do `take(1)` here?

Reply:
@estrada9166 can you add a comment w/ a link to where you found that this was needed?